packages/prisma/schema.prisma

```diff
@@ -655,7 +659,9 @@ enum WorkflowMethods {
 
 model VerifiedNumber {
   id          Int    @id @default(autoincrement())
-  userId      Int
-  user        User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId      Int?
+  user        User?  @relation(fields: [userId], references: [id], onDelete: Cascade)
+  teamId      Int?
```

Comment:
when a number on a team workflow gets verified, the number is verified for the whole team and not just the user who verified the number